processing/requirements.txt

```diff
@@ -54,6 +54,7 @@ tzlocal==2.1              # via dateparser
 urllib3==1.25.10          # via botocore, requests
 w3lib==1.22.0             # via itemloaders, parsel, scrapy
 zope.interface==5.1.0     # via scrapy, twisted
+chompjs==1.0.16
```

Comment:
Could you install `pip-tools` in your environment and then:
1. add `chompjs` to `requirements.in`
2. run `pip-compile requirements.in`
3. commit the modified `requirements.in` and `requirements.txt`

That way we can keep a more readable dependencies file (`requirements.in`) and a fixed dependency tree for an easy reproducible build (`requirements.txt`)